toolz/_signatures.py

```diff
@@ -642,7 +642,7 @@ def signature_or_spec(func):
         try:
             return inspect.signature(func)
         except (ValueError, TypeError) as e:
```

Comment:
This patch makes `e` unused variable.
```suggestion
        except (ValueError, TypeError):
```